dipy/reconst/gqi.py

```diff
@@ -109,11 +110,13 @@ def odf(self, sphere):
         self.gqi_vector = self.model.cache_get('gqi_vector', key=sphere)
         if self.gqi_vector is None:
             if self.model.method == 'gqi2':
-                H=squared_radial_component
-                #print self.gqi_vector.shape
-                self.gqi_vector = np.real(H(np.dot(self.model.b_vector,                                         sphere.vertices.T) * self.model.Lambda))
+                H = squared_radial_component
+                # print self.gqi_vector.shape
+                self.gqi_vector = np.real(H(np.dot(self.model.b_vector,
+                                                   sphere.vertices.T) * self.model.Lambda))
             if self.model.method == 'standard':
-                self.gqi_vector = np.real(np.sinc(np.dot(self.model.b_vector,                                   sphere.vertices.T) * self.model.Lambda / np.pi))
+                self.gqi_vector = np.real(np.sinc(np.dot(self.model.b_vector,
+                                                         sphere.vertices.T) * self.model.Lambda / np.pi))
```

Comment:
This line is still too long (>80 characters). Do you have a text editor that tells you about PEP8 issues? I use Atom with a PEP8 linter: https://atom.io/packages/linter-pep8